.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
         if: steps.check_rerun.outputs.rerun == 'false'
```

Comment:
- Instead of using a sed command to remove trailing newlines, consider using a more efficient method like trimming the string in bash.